### PR TITLE
chain/bandoracled2: initial hacky implementation

### PR DESCRIPTION
**Pull request by sorawit**
Initial hacky version of bandoracled. Improvements over v1
- Read raw data requests from events
- Use subscribe to get new tx. Not polling.

### CODE REVIEW

File: chain/cmd/bandoracled2/execute.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+func BroadCastMsgs(client *rpchttp.HTTP, key keyring.Info, msgs []sdk.Msg) {
+	// TODO: Make this a queue. Make it better.
+	cliCtx := sdkCtx.CLIContext{Client: client}
+	acc, err := auth.NewAccountRetriever(appCodec, cliCtx).GetAccount(key.GetAddress())
+	if err != nil {
+		fmt.Println("ERR1", err)
+		return
+	}
+
+	out, err := auth.NewTxBuilder(
```
Comment (taobun) [inline chain/cmd/bandoracled2/execute.go:38]: Add TODO config gas limit and gas price

File: chain/cmd/bandoracled2/execute.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		return
+	}
+
+	res, err := cliCtx.BroadcastTxCommit(out)
+	if err != nil {
+		fmt.Println("ERR3", err)
+		return
+	}
+
+	fmt.Println("EZ", res)
```
Comment (taobun) [inline chain/cmd/bandoracled2/execute.go]: Remove or log tx hash instead

File: chain/cmd/bandoracled2/execute.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+func init() {
+	authclient.Codec = appCodec
+}
+
+func BroadCastMsgs(client *rpchttp.HTTP, key keyring.Info, msgs []sdk.Msg) {
+	// TODO: Make this a queue. Make it better.
+	cliCtx := sdkCtx.CLIContext{Client: client}
+	acc, err := auth.NewAccountRetriever(appCodec, cliCtx).GetAccount(key.GetAddress())
+	if err != nil {
+		fmt.Println("ERR1", err)
```
Comment (taobun) [inline chain/cmd/bandoracled2/execute.go]: Use logger

File: chain/cmd/bandoracled2/handler.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+		return
+	}
+	if len(dataSourceIDs) != len(calldataList) {
+		logger.Error("😱 Request #%d: inconsistent data source count and calldata count", id)
+		return
+	}
+
+	reports := make([]oracle.RawReport, 0)
+	for idx, _ := range dataSourceIDs {
+		dataSourceID, err := strconv.Atoi(dataSourceIDs[idx])
```
Comment (taobun) [inline chain/cmd/bandoracled2/handler.go:74]: TODO: Make all of this to function and use go routine.

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]

**Review by Benzbeeb — APPROVED** [review]

**Pull request merged**